documentation/debugging-staticanalysis-errors.md

```diff
@@ -0,0 +1,47 @@
+- On the status page in Jenkins, under the Build Artifacts: the relevant files are BreakingChangeIssues.csv, SignatureIssues.csv, and/or HelpIssues.csv.
+- On the status page in Jenkins, click Build Artifacts then navigate to src/Package.  You will see BreakingChangeIssues.csv, SignatureIssues.csv, and/or HelpIssues.csv.
+
+Locally, the StaticAnalysis report will show up under Azure-PowerShell/src/Package. You will see BreakingChangeIssues.csv, SignatureIssues.csv, and/or HelpIssues.csv.  You can generate these files by running
+```
+msbuild build.proj
+```
+
+## Types of StaticAnalysis Errors
+The three most common Static Analysis errors are breaking changes, signature issues, and help issues.  To figure out which type of error is causing the build failure, open each of the relevant .csv files (if the .csv file does not exist, there is no violation detected).  Any issue marked with severity 0 or 1 must be resolved in order for the build to pass.
+
+### Breaking Changes
+If you make a change that could cause a breaking change, it will be listed in BreakingChangeIssues.csv.  Please look at each of these errors, and if they do indeed introduce a breaking change in a non-breaking change release, please revert the change that caused this violation.  Sometimes the error listed in the .csv file can be a false positive (for example, if you change a parameter attribute to span all parameter sets rather than individual parameter sets).  Please read the error thoroughly and examine the relevant code before deciding that an error is a false positive, and contact the Azure PowerShell team if you have questions.  If you are releasing a preview module, are releasing during a breaking change release, or have determined that the error is a false positive, please follow these instructions:
+- Navigate to the [BreakingChangeIssues.csv file](https://github.com/Azure/azure-powershell/blob/preview/tools/StaticAnalysis/Exceptions/BreakingChangeIssues.csv) in our repo.
+- Copy each of the errors you would like to suppress directly from the BreakingChangeIssues.csv file output from the Jenkins build, and paste them at the bottom of the BreakingChangeIssues.csv file in our repo.
+- Push the changes to the .csv file and ensure they no longer show up in the BreakingChangeIssues.csv file output from the Jenkins build.
+
+We take breaking changes very seriously, so please be mindful about the violations that you suppress in our repo.
+
+### Signature Issues
+Signature issues occur when your cmdlets do not follow PowerShell standards.  Please check [this page](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines) to ensure you are following PowerShell guidelines.  Issues with severity 0 or 1 must be addressed, while issues with severity 2 are advisory.  If you have an issue with severity 0 or 1 that have been approved by the Azure PowerShell team, you can suppress them following these steps:
```

Comment:
@maddieclayton nit: _If you have an issue with severity 0 or 1 that **has** been approved by the Azure PowerShell team..._

Reply:
Done